networks/suzuka/suzuka-full-node/src/main.rs

```diff
@@ -0,0 +1,31 @@
+use anyhow::Context;
+use monza_full_node::{
+    MonzaFullNode,
+    partial::MonzaPartialFullNode,
+};
+
+#[tokio::main]
+async fn main() -> Result<(), anyhow::Error> {
+
+    #[cfg(feature = "logging")]
+    {
+        use tracing_subscriber::EnvFilter;
+
+        tracing_subscriber::fmt()
+            .with_env_filter(EnvFilter::try_from_default_env()
+                             .unwrap_or_else(|_| EnvFilter::new("info")))
+            .init();
+
+    }
+
+    let executor = MonzaPartialFullNode::try_from_env().await.context(
```

Comment:
```suggestion
    let executor = SuzukaPartialFullNode::try_from_env().await.context(
```

Also let's find a better name for this struct. `PartialFull` is inherently confusing. How about just `SuzukaPartialNode`?